README.md

```diff
@@ -65,29 +66,33 @@ The following provides instructions for how to build docs locally for visualizat
 4. Install MyST Parser `pip install myst_parser`
 5. CD into the docs directory `cd [path to this repo]/docs`
 6. Run `make html`
-7. This will generate HTML in docs/build/html. Setup a web server with the web root as docs/build/html or open docs/build/html/index.html in a browser.
- 
+7. This will generate HTML in `docs/build/html`. Setup a web server with the web root as `docs/build/html` or open `docs/build/html/index.html` in a browser.
+
 ### Vale
+
 Before pushing, run Vale and address suggestions and errors as applicable.
-1. Install [`vale`][Vale] 
+
+1. Install [`vale`][Vale]
 2. `vale .`
```

Comment:
I'd recommend here that we suggest they use VSCode which has the Vale extension configured. As they save their work, it'll underline/highlight issues. See the write up I made here: https://github.com/mautic/mautic-community-handbook/blob/main/docs/contributing/contributing_docs_rst.rst - perhaps we can link to this in the future, at the moment we're still in the process of converting the handbook over to Read the Docs so it's not live anywhere yet.

Reply:
Does something like this work?

> To work with Vale, we recommend using VSCode as it has the Vale extension configured. 

And then, remove the instructions here. Once the handbook is done, we can link it here.

Any thoughts?

Reply:
You do need to install Vale first on your local computer, but then using VSCode works very well as we have all the VSCode extensions pre-defined. So maybe something like:

Note: We recommend using VSCode as this repository has all the extensions pre-defined. Other editors will probably have their own Vale linter plugins which help to identify problems as you type and suggest fixes.

You can use the note admonition: https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html#notes

Reply:
As this README is written in Markdown and live on GitHub, should we use the [GitHub admonition](https://github.com/orgs/community/discussions/16925) instead?